### PR TITLE
chore: rename the task of SyncKubelet

### PR DESCRIPTION
**Pull request by kiragoo**
### What type of PR is this?

/kind cleanup

### Which issue(s) this PR fixes:

Fixes #

### Special notes for reviewers:
```
According to the definition of function `Execute`  about `SyncKubelet`, the task just changes the owner of `kubelet` binary. In addition, we can see the task of `SyncKubeBinary` do the work of syncing `kubelet` already.
As a result, maybe the name of  the task `SyncKubelet` changed to `ChownKubelet` is better for developers to understand.
```

### Does this PR introduced a user-facing change?

```release-note
None
```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

File: cmd/kk/pkg/kubernetes/tasks.go [hunk 1]
```diff
@@ -154,11 +154,11 @@ func SyncKubeBinaries(runtime connector.Runtime, binariesMap map[string]*files.K
...
 
-type SyncKubelet struct {
+type ChownKubelet struct {
 	common.KubeAction
 }
 
-func (s *SyncKubelet) Execute(runtime connector.Runtime) error {
+func (c *ChownKubelet) Execute(runtime connector.Runtime) error {
 	if _, err := runtime.GetRunner().SudoCmd("chmod +x /usr/local/bin/kubelet", false); err != nil {
 		return errors.Wrap(errors.WithStack(err), "sync kubelet service failed")
```
Comment (pixiake) [inline cmd/kk/pkg/kubernetes/tasks.go:163]: Maybe this error log can also be modified.

File: cmd/kk/pkg/kubernetes/tasks.go [hunk 2]
```diff
@@ -154,11 +154,11 @@ func SyncKubeBinaries(runtime connector.Runtime, binariesMap map[string]*files.K
 	return nil
 }
 
-type SyncKubelet struct {
+type ChownKubelet struct {
 	common.KubeAction
 }
 
-func (s *SyncKubelet) Execute(runtime connector.Runtime) error {
+func (c *ChownKubelet) Execute(runtime connector.Runtime) error {
```
Comment (pixiake) [inline cmd/kk/pkg/kubernetes/tasks.go:161]: `chown` may need to be replaced with `chmod`.

Comment (kiragoo) [reply cmd/kk/pkg/kubernetes/tasks.go:161]: Ok, I'll re-push the pr after modifying the content.

Comment (kiragoo) [reply cmd/kk/pkg/kubernetes/tasks.go:161]: > `chown` may need to be replaced with `chmod`.

Ok, I'll re-push the pr after modifying the content.

File: cmd/kk/pkg/kubernetes/tasks.go (hunk 1 above)
Comment (kiragoo) [reply cmd/kk/pkg/kubernetes/tasks.go:163]: > Maybe this error log can also be modified.

Sorry for that, I'll re-push the pr to complete the log of error.

**Comment by kiragoo** [comment]
> @kiragoo Thanks for this contribution !

Thx for your review! 😀 It's my pleasure.

**Pull request merged**